### ISSUE TITLE
Add possibility to disable the vendor prefixer (Vite)

### ISSUE THREAD

**Issue by functionalDev** [Anber/wyw-in-js#77]
## Describe the feature

Adding the option to disable vendor prefixes for the generated css. The default stylis-preprocessor always adds vendor prefixes to the css.  

## Motivation

Serve less CSS. AMP (accelerated mobile pages) have a limit to the css they are allowed to contain. Removing vendor prefixes was one way we reduced our css with styled component.

## Possible implementations

You already did a great job making everything flexible and everything is already in place. You just need to accept the flag `prefixer` in the plugin and pass it in to the transform function.

**Comment by petr001** [comment]
Yep, `prefixer` is there, it's just not passed from higher functions and options object:

https://github.com/Anber/wyw-in-js/blob/main/packages/transform/src/transform/generators/createStylisPreprocessor.ts#L430

Vendor prefixes are obsolete, making CSS 25% bigger.
